0110    javascript-loops-practice/includes-seven.js

```diff
@@ -0,0 +1,16 @@
+/* exported includesSeven */
+function includesSeven(array) {
+  let boolean = false;
+
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] === 7) {
+      boolean = true;
+      break;
```

Comment:
another way of writing this would be to just `return true` inside of this if conditional. That way you don't even need a local variable.

If you iterate through the entire loop, you can just `return false` at the end of the function code block. 

This works because `return` statements exit out of the function at the point it is encountered.